src/main/kotlin/com/cultureamp/eventsourcing/jsonb.kt

```diff
@@ -1,27 +1,33 @@
 package com.cultureamp.eventsourcing
 
-
 import org.jetbrains.exposed.sql.Column
 import org.jetbrains.exposed.sql.StringColumnType
 import org.jetbrains.exposed.sql.Table
 import org.jetbrains.exposed.sql.statements.api.PreparedStatementApi
-
-/**
- * Modified from: https://gist.github.com/quangIO/a623b5caa53c703e252d858f7a806919
- */
+import org.postgresql.util.PGobject
 
 fun Table.jsonb(name: String): Column<String> =
     registerColumn(name, Jsonb())
 
-
 private class Jsonb : StringColumnType() {
     override fun sqlType() = "jsonb"
 
     override fun setParameter(stmt: PreparedStatementApi, index: Int, value: Any?) {
-        stmt[index] = value as String
+        if (value is String) {
+            val pgObject = PGobject()
+            pgObject.type = "jsonb"
+            pgObject.value = value
+            stmt[index] = pgObject
+        } else {
+            stmt[index] = value!!
```

Comment:
does stmt only accept not nullables? I wonder if there's another way to set null eg `stmt.setNull(index)`